server/routes/api/auth.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express');
+const router = express.Router();
+const bcrypt = require('bcrypt');
+
+// User Model
+const User = require('../../models/user');
+
+// POST User Login
+router.post('/login', login);
+
+function login(req, res) {
+  const email = req.body.email;
```

Comment:
You probably don't need to create constants out of these if you're just tossing them into `.authenticate()`.